frontend/src/layouts/widget/UserMenu.jsx

```diff
@@ -0,0 +1,51 @@
+import React from "react";
+import {
+  UncontrolledDropdown,
+  DropdownToggle,
+  DropdownMenu,
+  DropdownItem,
+} from "reactstrap";
+import { FiLogOut } from "react-icons/fi";
+import { IoMdSettings } from "react-icons/io";
+
+import { UserBubble, DropdownNavLink } from "@certego/certego-ui";
+
+import { useAuthStore } from "../../stores";
+
+const djangoAdminLink = (
```

Comment:
This function is very short and used only once, you could avoid to define a function and put this code directly where you need to render it